python/servo/command_base.py

```diff
@@ -553,6 +554,11 @@ def common_command_arguments(build_configuration=False, build_type=False, binary
                     help='Build for Android. If --target is not specified, this '
                          'will choose a default target architecture.',
                 ),
+                CommandArgument(
+                    '--ohos', default=None, action='store_true',
```

Comment:
What benefit does adding a new argument give us? In principle the target triple already tells us that we are targetting ohos.

Reply:
It is just an alias to build the default architecture for the platform. The only value is the UX for the user, which is debatable, but we have it for android, so I've added for OH as well.

Reply:
Considering that the long term plan is to remove mach, I'm not convinced having users get used to convenience options that "cargo build" does not have is desirable.

Reply:
Fair,  but I don't see mach being replaced very soon, so I don't see the harm in having a consistent UX while we still depend upon it.

Reply:
mach will not be removed, but our plan is to make `mach build/check/cargo-*` commands more thin (ideally just cargo commands aliases with some options aliases).